plugins/save/plugin.js

```diff
@@ -44,8 +44,9 @@
 			if ( editor.elementMode != CKEDITOR.ELEMENT_MODE_REPLACE )
 				return;
 
+			saveCmd.startDisabled = !( editor.element.$.form );
 			var command = editor.addCommand( pluginName, saveCmd );
-			command.modes = { wysiwyg: !!( editor.element.$.form ) };
+			command.modes = { wysiwyg: 1,source: 1 };
```

Comment:
This property OTOH should be added to template (`saveCmd`) as it's shared across all commands created using that template.